### DISCUSSION TITLE
Manually update Media Progress Time & Bar

### DISCUSSION THREAD

**Discussion by mazowo** [VolcanicArts/VRCOSC#143] [Ideas]
I know that some media sources like browsers don't update the time properly, however a possible solution for this could be to manually add the DeltaTime if the program detects a playing state, but no increase in time.

Note, that the following is only an example. I'm not versed in c# and it's obviously not tested,
`MediaVariable.Time` might not be an integer and `deltaTime` would need to be calculated somehow.
```c#
private int previousTime = 0;

private int getMediaTime()
{
  private int mediaTime = MediaVariable.Time;
  if (mediaTime == previousTime)
  {
    mediaTime = mediaTime + deltaTime; //Time since last update
  }
  
  previousTime = mediaTime;

  return mediaTime;
}
```

**Comment by VolcanicArts** [comment]
https://github.com/VolcanicArts/VRCOSC/commit/2158de1dcff0397eef9c2e209a84b58ac34250a6
